content/5_Plat/RURQ.mdx

```diff
@@ -152,135 +152,127 @@ Given the mechanics behind the `pushdown` function, all you need is a regular
 range-sum segment tree to solve the question.
 
 ```cpp
-
-#include <iostream>
-using ll = long long;
+#include <bits/stdc++.h>
 using namespace std;
+using ll = long long;
 
-const int maxN = 2e5 + 5;
-
-int N, Q;
-int a[maxN];
+enum QueryType { ADD, SET, NONE };
```

Comment:
ayo a "none" query type?

Reply:
technically its not needed but i think its good b/c it specifies when there's no lazy update to be performed